src/Microsoft.DotNet.ProjectJsonMigration/SupportedPackageVersions.cs

```diff
@@ -191,6 +191,30 @@ public SupportedPackageVersions()
                     {
                         new PackageDependencyInfo
                         {
+                            Name = "Microsoft.VisualStudio.Web.BrowserLink.Loader",
+                            Version = "[1.1.0-*,)"
+                        },
+                        new PackageDependencyInfo
+                        {
+                            Name = "Microsoft.VisualStudio.Web.BrowserLink",
+                            Version = "1.1.0"
+                        }
+                    },
+                    {
+                        new PackageDependencyInfo
+                        {
+                            Name = "Microsoft.VisualStudio.Web.BrowserLink.Loader",
+                            Version = "[1.0.0-*,1.1.0)"
```

Comment:
Shouldn't this look like "[14.0.0, 14.1.0)" rather than ""[1.0.0-*,1.1.0)"? I believe the Dev14 versions of this package were versioned with 14.* -- see http://www.nuget.org/packages/Microsoft.VisualStudio.Web.BrowserLink.Loader